src/components/Calendar.tsx

```diff
@@ -39,11 +49,15 @@ export default function CalendarPage({ isAuthorized, data, logOut }: ComponentPr
           <p className='text-lg'>Quarter Moon 🌖</p>
```

Comment:
does this piece need to be dynamic instead of hard coded with `quarter moon`? (Maybe this change is on Saki's branch?)

Reply:
I think that's coming up on sakis branch!